### DISCUSSION TITLE
[Fundamentos do ASP.NET 6]-[Finalizando o Crud de categorias]

### DISCUSSION THREAD

**Discussion by diefersonlopes** [balta-io/forum#252] [Tira Dúvidas] [answered]
Olá!
Estou com um erro no momento de dar um "PUT" nas minhas categorias no curso de "Fundamentos do ASP.NET 6", etapa "CRUD e Entity Framework", no tópico "Finalizando o Crud de categorias". Acontece que não estou conseguindo alterar os dados, adicionei um try/catch e a excessão gerada é a:
![image](https://user-images.githubusercontent.com/122637123/216783959-403a29ba-d693-44a7-a83d-9320a75d8fad.png)

Obrigado desde já!

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @diefersonlopes, como vai?
Qual código está sendo executado?

**Reply by diefersonlopes** [reply]
Minha controller:
![image](https://user-images.githubusercontent.com/122637123/217060463-34346d9b-a4cb-402d-9775-074822402faa.png)

Meu model:
![image](https://user-images.githubusercontent.com/122637123/217060609-da95ef39-bd78-40fb-b4fa-0ddad3397a4f.png)

Meu mapping:
![image](https://user-images.githubusercontent.com/122637123/217060738-ad905965-4a3b-43ac-8d89-7ecaf8d603c7.png)

Meu json:
![image](https://user-images.githubusercontent.com/122637123/217060876-bf5af6db-fae5-450d-8f48-cf47eb3396ff.png)

**Reply by andrebaltieri** [reply]
Boa noite, @diefersonlopes, como vai?
Você consegue debugar para ver se está realmente chegando no método POST?
Caso esteja, consegue garantir que não há outra categoria já cadastrada com o **slug** `frontend`?
